src/components/shared/FormField/FormField.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen } from "@testing-library/react";
+import { FormField } from ".";
+
+describe("FormField", () => {
+  it("renders input with label", () => {
+    render(<FormField name="email" />);
+
+    const input = screen.getByRole("textbox", { name: /email/i });
```

Comment:
New line before assertions would be nice.
It clearly splits the test into `Arrange/Act/Assert` or `Given/When/Then` pattern